asabiris/handlers/kafkahandler.py

```diff
@@ -56,11 +56,21 @@ def __init__(self, app, service_name="KafkaHandler"):
 		)
 
 	async def initialize(self, app):
-		try:
-			await self.Consumer.start()
-		except aiokafka.errors.KafkaConnectionError as e:
-			L.warning("No connection to Kafka established. Stopping the app... {}".format(e))
+		max_retries = 5
+		delay = 5  # Initial delay in seconds
+		for attempt in range(max_retries):
+			try:
+				await self.Consumer.start()
+				break
+			except aiokafka.errors.KafkaConnectionError as e:
+				L.warning("No connection to Kafka established. Attempt {} of {}. Retrying in {} seconds... {}".format(
+					attempt + 1, max_retries, delay, e))
+				await asyncio.sleep(delay)
+				delay *= 2  # Exponential backoff
```

Comment:
This is good but there must a be upper limit (5 minutes for example) to prevent too long retries ...